CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where searching for unlinked files would include the current library's .bib file [#9735](https://github.com/JabRef/jabref/issues/9735)
 - We fixed an issue where it was no longer possible to connect to a shared mysql database due to an exception [#9761](https://github.com/JabRef/jabref/issues/9761)
 - We fixed the citation key generation for (`[authors]`, `[authshort]`, `[authorsAlpha]`, `authIniN`, `authEtAl`, `auth.etal`)[https://docs.jabref.org/setup/citationkeypatterns#special-field-markers] to handle `and others` properly. [koppor#626](https://github.com/koppor/jabref/issues/626)
+- We fixed an issue regarding recording redundant prefixes in search history, by attaching a change listener on the focus property of search box. [#9685](https://github.com/JabRef/jabref/issues/9685)
```

Comment:
```suggestion
- We fixed an issue regarding recording redundant prefixes in search history [#9685](https://github.com/JabRef/jabref/issues/9685)
```

The changelog is intended for the "normal end user"  so no  technical details necessary